src/Storefront/Resources/views/storefront/layout/cookie/cookie-permission.html.twig

```diff
@@ -34,6 +43,18 @@
                                 </button>
                             </span>
                         {% endblock %}
+
+                        {% block layout_cookie_permission_inner_button_accept_all %}
+                            {% if context.salesChannel.cookieAcceptAllActive %}
+                                <span class="js-cookie-accept-all">
+                                        <button
```

Comment:
I think it would be better to put the class `js-cookie-accept-all` and the click event directly on the button - rather than putting it on a wrapping `span` element. I am not sure why the other buttons have an extra `span` element. Maybe you can help here @bschulzebaek?

I'm also fine with sticking to this pattern for now.

Reply:
I guess, the `span` element was used for styling. But yes, the button should always get the click-event.